pandas/core/frame.py

```diff
@@ -2726,7 +2726,8 @@ def duplicated(self, subset=None, take_last=False):
             Only consider certain columns for identifying duplicates, by
             default use all of the columns
         take_last : boolean, default False
-            Take the last observed row in a row. Defaults to the first row
+             For a set of distinct duplicate rows, flag all but the last row as duplicated. 
+             Default is for all but the first row to be flagged
```

Comment:
Thank you. Amended.

On 6 January 2015 at 16:53, Joris Van den Bossche <notifications@github.com>
wrote:

> In pandas/core/frame.py
> <https://github.com/pydata/pandas/pull/9203#discussion-diff-22555271>:
>
> > @@ -2726,7 +2726,8 @@ def duplicated(self, subset=None, take_last=False):
> >              Only consider certain columns for identifying duplicates, by
> >              default use all of the columns
> >          take_last : boolean, default False
> > -            Take the last observed row in a row. Defaults to the first row
> > +             For a set of distinct duplicate rows, flag all but the last row as duplicated.
> > +             Default is for all but the first row to be flagged
>
> Very small detail: I think your new line is one space off in indentation
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/9203/files#r22555271>.
>